Money_Counter_JB.py

```diff
@@ -0,0 +1,44 @@
+# This programme counts British coins based on their weight
+
+# Started with dictionary of UK coin weights
+
+coindict = {
+    "1": 3.56,
+    "2": 7.12,
+    "5": 3.25,
+    "10": 6.5,
+    "20": 5.0,
+    "50": 8.0
+    }
+
+while True:     # Put this eternal loop in so I don't have to keep restarting it
+
+    print()     # Blank lines for aesthetics
+
+    objective = input("\nWould you like to \n\n(a) weigh coins \nor \n(b) fill a coin bag?\n\n " ) # First choice: find the value of a batch of coins or fill a bag correctly?
```

Comment:
Two points here. Firstly, you can space out the code here to make it a bit more readable (say you leave this and come back to it in a few months and you need a bit of help understanding what is going on here and the code being distributed over a few lines makes life that bit easier). Secondly, you can use your variables to populate the text using [f strings](https://realpython.com/python-f-strings/), an easy and awesome way to substitute values in a string.
```suggestion
objective = input(
        f"\nWould you like to \n\n({weigh_coins_option}) weigh coins \nor \n({fill_bag_option}) fill a coin bag?\n\n "
)
```